cmd/kube-scheduler/app/server.go

```diff
@@ -348,7 +355,7 @@ func installMetricHandler(pathRecorderMux *mux.PathRecorderMux, informers inform
 // embed the metrics handler.
 // TODO: healthz check is deprecated, please use livez and readyz instead. Will be removed in the future.
 func newHealthEndpointsAndMetricsHandler(config *kubeschedulerconfig.KubeSchedulerConfiguration, informers informers.SharedInformerFactory, isLeader func() bool, healthzChecks, readyzChecks []healthz.HealthChecker) http.Handler {
```

Comment:
Statusz does not belongs to healthz heandlers group. Can you move it either outside, or rename the method to something like installEndpointHandlers?

Reply:
I follow the flagz implementation here (https://github.com/kubernetes/kubernetes/pull/128818/files) and move it outside